From c2ebf45e455b6ef20a37a038e6e1d549b2e1fb3c Mon Sep 17 00:00:00 2001
From: "Cui, Yuxin" <yuxin.cui@intel.com>
Date: Fri, 25 Feb 2022 15:20:45 +0800
Subject: [PATCH] Modify checkDumpPermission for network_stack

Add android.permission.DUMP into Exception msg

Tracked-On: OAM-100531
Change-Id: I14756107cca5658230ebb42846dc0651fba31a2e
Signed-off-by: Cui, Yuxin <yuxin.cui@intel.com>
---
 src/com/android/server/util/PermissionUtil.java | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/com/android/server/util/PermissionUtil.java b/src/com/android/server/util/PermissionUtil.java
index 3dff7151..08db60e4 100644
--- a/src/com/android/server/util/PermissionUtil.java
+++ b/src/com/android/server/util/PermissionUtil.java
@@ -75,7 +75,7 @@ public final class PermissionUtil {
         final int caller = getCallingUid();
         if (caller != Process.myUid() && caller != Process.SYSTEM_UID && caller != Process.ROOT_UID
                 && caller != Process.SHELL_UID) {
-            throw new SecurityException("No dump permissions for caller: " + caller);
+            throw new SecurityException("No android.permission.DUMP permissions for caller: " + caller);
         }
     }
 
-- 
2.25.1

